checker/lib/README.developers

```diff
@@ -0,0 +1,11 @@
+Please read the README file in this directory first, as it is the file
+that is shipped in the distribution and is intended for users of the
+Checker Framework.
+
+When adding a .jar file to this directory, update the list in README and also
+the list in ../manual/annotating-libraries.tex .
```

Comment:
The list in that file is commented out, and it's not reinstated by this pull request.